libs/redis/langchain_redis/cache.py

```diff
@@ -112,13 +120,19 @@ def __init__(
         redis: Optional[Redis] = None,
     ):
         self.redis = redis or Redis.from_url(redis_url)
+        try:
+            self.redis.client_setinfo("LIB-NAME", __with_redis_py_only_lib_name__)  # type: ignore
```

Comment:
@bsbodden This command should reflect RedisVL + LangChain, not just redis py? Since we are using the RedisVL SemanticCache it's already utilizing RedisVL and the base SearchIndex class here

Reply:
If I am understanding, I think the goal is to be able to make sure we are properly highlighting langchain, and since the semantic cache doesn't give you a way to modify the core SearchIndex `lib-name` param (yet), we need to workaround. Is that right?

Reply:
@tylerhutcherson yes, the libname in `RedisSemanticCache` needs to change to the RedisVL one, in the `RedisCache` class we do not use RedisVL, so I felt that reporting it using RedisVL was not accurate, WDYT?

Reply:
Reporting all as RedisVL now